GenericDockerfile

```diff
@@ -29,7 +29,7 @@ LABEL org.label-schema.description="System configuration data for the CARMA Plat
 LABEL org.label-schema.vendor="Leidos"
 LABEL org.label-schema.version=${VERSION}
 LABEL org.label-schema.url="https://highways.dot.gov/research/research-programs/operations/CARMA"
-LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/CARMAPlatform"
+LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-platform"
```

Comment:
Huh, didn't even know that said CARMAPlatform. Should be carma-config, we want it to reflect the repo it came from